signer/src/proto/convert.rs

```diff
@@ -1089,15 +1093,30 @@ impl From<WstsMessage> for proto::WstsMessage {
                 proto::wsts_message::Inner::SignatureShareResponse(inner.into())
             }
         };
+
         proto::WstsMessage {
-            txid: Some(BitcoinTxId::from(value.txid).into()),
+            txid: match value.id {
+                WstsMessageId::BitcoinTxid(txid) => {
+                    Some(proto::BitcoinTxid::from(BitcoinTxId::from(txid)))
+                }
+                WstsMessageId::RotateKey(_) => None,
+            },
+            id: Some(match value.id {
+                WstsMessageId::BitcoinTxid(txid) => {
+                    wsts_message::Id::IdBitcoinTxid(proto::BitcoinTxid {
+                        txid: Some(proto::Uint256::from(BitcoinTxId::from(txid).into_bytes())),
+                    })
+                }
+                WstsMessageId::RotateKey(pubkey) => wsts_message::Id::IdRotateKey(pubkey.into()),
+            }),
```

Comment:
This is for backwards compatibility. When serializing a `message:WstsMessage` to a `proto::WstsMessage`, it will set both the `txid` and the `id` to a `proto::BitcoinTxid` if the source id type is `BitcoinTxid` (the only current case until #1285 merges and DKG runs).